plugins/azure/cosmosdb/cosmosdbLocalAuth.js

```diff
@@ -0,0 +1,53 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Cosmos DB Local Authentication Disabled',
+    category: 'Cosmos DB',
+    domain: 'Databases',
+    severity: 'Low',
+    description: 'Ensures that local authentication is disabled for Cosmos DB accounts.',
+    more_info: 'For enforcing orce clients to connect to Azure Cosmos DB through role-based access control exclusivelyenhanced security, centralized identity management, and seamless integration with Azure\'s authentication and authorization services, it is recommended to rely on Azure Active Directory (Azure AD) and disable local authentication for Azure Cosmos DB accounts. It enforces that client connect to Azure Cosmos DB through role-based access only.',
```

Comment:
```suggestion
    more_info: 'For enhanced security, centralized identity management and seamless integration with Azure\'s authentication and authorization services, it is recommended to rely on Azure Active Directory (Azure AD) and disable local authentication for Azure Cosmos DB accounts.',
```

not sure what the word should be here instead of 'orce', and maybe we can rephrase it like this